ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -271,6 +271,23 @@ echo "$(date) - waiting for clusteroperators to finish progressing..."
 oc wait clusteroperators --all --for=condition=Progressing=false --timeout=10m
 echo "$(date) - all clusteroperators are done progressing."
 
+# TODO: Temporary, move into the failure case block below before merging:
+if [[ -n "${TEST_REQUIRES_SSH-}" ]]; then
+	echo "Attempting to gather kubelet logs from each worker node via ssh bastion pod"
+	mkdir -p "${ARTIFACT_DIR}/worker-kubelet-logs/"
+	# We should have an ssh bastion pod per above. Now we use it to try to gather kubelet logs from each worker.
+	WORKER_IPS="$(oc --insecure-skip-tls-verify get machines -n openshift-machine-api -l machine.openshift.io/cluster-api-machine-role=worker -o 'jsonpath={.items[*].status.addresses[?(@.type=="InternalIP")].address}')"
+	echo "Found worker IPs: $WORKER_IPS"
+	ingress_host="$(oc get service --all-namespaces -l run=ssh-bastion -o go-template='{{ with (index (index .items 0).status.loadBalancer.ingress 0) }}{{ or .hostname .ip }}{{end}}')"
+	echo "Ingress host: $ingress_host"
+
+	for ip in $WORKER_IPS
+	do
+		echo "Gathering kubelet logs from ${ip}"
+	    	ssh -i "${KUBE_SSH_KEY_PATH}" -t -o StrictHostKeyChecking=no -o ProxyCommand="ssh -i ${KUBE_SSH_KEY_PATH} -A -o StrictHostKeyChecking=no -o ServerAliveInterval=30 -W %h:%p core@${ingress_host}" core@$ip "sudo journalctl -u kubelet --no-pager" > "${ARTIFACT_DIR}/worker-kubelet-logs/${ip}-kubelet.log"
```

Comment:
IIRC, when we gather logs on nodes via must gather, we don't filter by unit normally. If we filter to the kubelet unit, any dependency failures won't be picked up. Is it better to just take the whole journal and filter manually later?

Reply:
Yeah that seems fine to me, I'll switch it over.